From: coolsnowwolf <coolsnowwolf@gmail.com>
Subject: [PATCH] backports: pass extra compiler flags on Linux 6.18

Linux 6.18 no longer consumes EXTRA_CFLAGS in the kernel Kbuild.
Merge the backports include paths into KCFLAGS for module builds.

Signed-off-by: coolsnowwolf <coolsnowwolf@gmail.com>
---
--- a/Makefile.build
+++ b/Makefile.build
@@ -10,7 +10,8 @@
 
 .PHONY: modules
 modules:
-	@$(MAKE) -C $(KLIB_BUILD) M=$(BACKPORT_DIR) modules
+	@$(MAKE) -C $(KLIB_BUILD) M=$(BACKPORT_DIR) \
+		KCFLAGS="$(KCFLAGS) $(EXTRA_CFLAGS)" modules
 
 .PHONY: clean
 clean:
